CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/BaseTest/AccessibilityIdentifier/ConfigurationScreenAccessibilityIdentifier.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ConfigurationScreenAccessibilityIdentifier.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+enum ConfigurationScreenAccessibilityIdentifier:String{
+    case closeButton = "closeButtonIdentifier"
+    case qrCodeScanSwitch = "qrCodeScanSwitchIdentifier"
+    case qrCodeScanOnlySwitch = "qrCodeScanOnlySwitchIdentifier"
+    case multiPageSwitch = "multipageSwitch"
```

Comment:
[clarification]: I think this should be`multipageSwitchIdentifier`.

Although I have mix fillings with this way of naming `ConfigurationScreenAccessibilityIdentifier` and each case with `identifier` in the end. It sounds of duplication and can lead to long names.

Reply:
We can discuss this in our meeting on Friday, 05-07-2024